stable/pomerium/templates/authenticate-deployment.yaml

```diff
@@ -0,0 +1,141 @@
+{{- $secretName := default (include "pomerium.fullname" .) .Values.config.existingSecret }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "pomerium.name" . }}-{{ .Values.authenticate.name }}
+    chart: {{ template "pomerium.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
```

Comment:
The common labels changed awhile back due to recommendations from Kubernetes. Can you update them per the docs at https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata. This applies to all the objects